### PR TITLE
ThreatConnect Feed

### PR DESCRIPTION
**Pull request by GalRabin**
**docker image should be replaced**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/21393
fixes: https://github.com/demisto/etc/issues/27629

## Description
Feed for ThreatConnect - as requested use can filter by owners.

## Screenshots
image

## Minimum version of Demisto
- [ ] 4.5.0
- [x] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/FeedThreatConnect/Integrations/FeedThreatConnect/FeedThreatConnect.py
Comment (yaakovi) [inline Packs/FeedThreatConnect/Integrations/FeedThreatConnect/FeedThreatConnect.py:49]:
```suggestion
    """Disable stdout in beginning and enable it in exit"""
```

**Comment by orhovy** [comment]
please add feedtags key to yml file and implement in the code.
you can see this pr for an example:
https://github.com/demisto/content/pull/8352

File: Packs/FeedThreatConnect/Integrations/FeedThreatConnect/FeedThreatConnect.py
Comment (yaakovi) [inline Packs/FeedThreatConnect/Integrations/FeedThreatConnect/FeedThreatConnect.py:115]:
```suggestion
            1. When importing TcEx, Print occurred therefor an error raised in the server due to not valid stdout.
```

File: Packs/FeedThreatConnect/Integrations/FeedThreatConnect/FeedThreatConnect.py
Comment (yaakovi) [inline Packs/FeedThreatConnect/Integrations/FeedThreatConnect/FeedThreatConnect.py:20]:
```suggestion
*** Error in demisto docker loop, when importing tcex module a print occurred therefor it's handled with context manager to suppress prints. 
```

**Review by yaakovi — COMMENTED** [review]
you added the `feedTags` parameter but you are not using it in the code

**Comment by GalRabin** [comment]
@kirbles19 
Hi,
I will appreciate your review on:
1. [Readme.md](https://github.com/demisto/content/blob/ThreatConnectIndicatorsFeed/Packs/FeedThreatConnect/Integrations/FeedThreatConnect/README.md)
2. [Description](https://github.com/demisto/content/blob/ThreatConnectIndicatorsFeed/Packs/FeedThreatConnect/Integrations/FeedThreatConnect/FeedThreatConnect_description.md)
3. [YAML](https://github.com/demisto/content/blob/ThreatConnectIndicatorsFeed/Packs/FeedThreatConnect/Integrations/FeedThreatConnect/FeedThreatConnect.yml)

**Comment by GalRabin** [comment]
> you added the `feedTags` parameter but you are not using it in the code

You right, @bakatzir added it, But I didn't fully understand the functionality, Now I am using it in the code as required.
Waiting for docs approval.

**Comment by kirbles19** [comment]
@GalRabin Done. Please regenerate the README from the updated YML and MD files.

**Comment by GalRabin** [comment]
> @GalRabin Done. Please regenerate the README from the updated YML and MD files.

Thanks done

File: Packs/FeedThreatConnect/Integrations/FeedThreatConnect/FeedThreatConnect.py
Comment (yaakovi) [inline Packs/FeedThreatConnect/Integrations/FeedThreatConnect/FeedThreatConnect.py:143]:
```suggestion
            Iterator: indicator objects.
```

File: Packs/FeedThreatConnect/Integrations/FeedThreatConnect/FeedThreatConnect.py
Comment (yaakovi) [inline Packs/FeedThreatConnect/Integrations/FeedThreatConnect/FeedThreatConnect.py:234]: otherwise the extra comma is redundant...
```suggestion
                    demisto.getParam("tc_api_path"),
                    )
```

**Review by yaakovi — APPROVED** [review]
pull from master

**Pull request merged**
